lib/rubocop/cop/rspec/factory_bot/attribute_defined_statically.rb

```diff
@@ -39,7 +39,7 @@ class AttributeDefinedStatically < Cop
 
           def on_block(node)
             attributes = factory_attributes(node) || []
-            attributes = [attributes] unless attributes.is_a?(Array)
+            attributes = [attributes] unless attributes.is_a?(Array) # rubocop:disable Style/ArrayCoercion, Lint/RedundantCopDisableDirective
```

Comment:
Why RedundantCopDisableDirective?

Reply:
Because otherwise, it triggers an offense in versions that still do not have this cop, e.g. in the current one

Reply:
Ah, ok. It’s a bit confusing :-)